bot/internal/bot/bot.go

```diff
@@ -137,7 +137,20 @@ func classifyChanges(e *env.Environment, files []github.PullRequestFile) (docs b
 	return docs, code, nil
 }
 
-func isLargePR(files []github.PullRequestFile) bool {
+func xlargeRequiresAdminApproval(files []github.PullRequestFile) bool {
+	return prSize(files) == xlarge
+}
+
+type sizeLabel string
+
+const (
+	small  sizeLabel = "size-sm"
+	medium sizeLabel = "size-md"
+	large  sizeLabel = "size-lg"
+	xlarge sizeLabel = "size-xl"
```

Comment:
Nit: What do you think about "namespacing" sizes like we do for other labels like "db/", etc.

```suggestion
	small  sizeLabel = "size/sm"
	medium sizeLabel = "size/md"
	large  sizeLabel = "size/lg"
	xlarge sizeLabel = "size/xl"
```

Reply:
Sure!